script/generate_sorted_docids_mapping.py

```diff
@@ -0,0 +1,36 @@
+ #!/usr/bin/env python
+
+import string
+from argparse import ArgumentParser
+
+def dump(output, sorted_docids):
+    fout = file(output, "w")
+    for newid in range(0, len(sorted_docids)):
+        # Note that sorted_docids[newid] is the currid.
+        fout.write(str(sorted_docids[newid]) + " " + str(newid) + "\n")
+
+
+def main():
+    parser = ArgumentParser(description='Take a document lexicon (e.g. \'.documents\' or\
```

Comment:
Let's use double quotes for either the string or the quotes inside, and get rid of the escape characters, e.g.,
`"Take a document lexicon (e.g. '.documents' ...`

Reply:
Same as above